server/utils/applications/managementInfoFromAppData.ts

```diff
@@ -28,4 +28,17 @@ const conditionalReleaseDateFromAppData = (application: Application): string =>
   return date || null
 }
 
-export { preferredAreasFromAppData, hdcEligibilityDateFromAppData, conditionalReleaseDateFromAppData }
+const telephoneNumberFromAppData = (application: Application): string => {
```

Comment:
I think the return value should be `string | null`? A bit surprised TS hasn't complained about that.

Reply:
hmm yeah good point